keps/sig-scheduling/4816-dra-prioritized-list/README.md

```diff
@@ -364,28 +598,183 @@ type DeviceRequest struct {
     // +oneOf=deviceRequestType
     DeviceClassName string
 
-    // FirstAvailableOf contains subrequests, exactly one of which must be satisfied
+    // FirstAvailable contains subrequests, exactly one of which must be satisfied
```

Comment:
at least one of which

Reply:
Fixed.

Reply:
"at least" implies that also more than one could be satisfied by the allocation results. That will never be the case.

How about "FirstAvailable contains subrequests, of which exactly one must be satisfied in order to satisfy this request."?

Reply:
There could be more than one that satisfies, but only one will be picked. Not sure how to phrase it, but the above doesn't sound right.

Reply:
Next try:

"FirstAvailable contains subrequests, of which exactly one will be satisfied by the scheduler to satisfy this request. It tries to satisfy them in the order in which they are listed here."

Reply:
Agree with that language. I have updated the PR.